.github/workflows/test.yaml

```diff
@@ -66,8 +66,35 @@ jobs:
             echo "include_test_modules=" >> $GITHUB_OUTPUT
             echo "exclude_test_modules=" >> $GITHUB_OUTPUT
             echo "exclude_tests=" >> $GITHUB_OUTPUT
+
+            if [[ "${{ github.event_name }}" == "schedule" ]]; then
```

Comment:
I think a comment explaining that this is currently checking for invalid triggers would be good, cause it looks a bit odd by itself.

Reply:
Done